helm/os-vim-driver/values.yaml

```diff
@@ -47,6 +47,11 @@ app:
         # timeout waiting for initial version check on Kafka producer/consumer initialisation
         # 5000ms is usually sufficient, increase if problems with NoBrokersAvailable occur
         api_version_auto_timeout_ms: 5000
+      adopt:
+        # Flag to override status check and allow adoption of a stack in any status
+        skip_status_check: False
+        # List of openstack stack status considered OK to adopt:
+        adoptable_status_values: ['CREATE_COMPLETE','ADOPT_COMPLETE','RESUME_COMPLETE','CHECK_COMPLETE','UPDATE_COMPLETE']  
```

Comment:
Should we provide a list of the allowable values here?

Reply:
added a comment with some further potential values